### PR TITLE
Finish Brake Light Code

### PR DESCRIPTION
**Pull request by rydzynskip**
Updated CAN information for the brake light

### CODE REVIEW

**Review by jamescd18 — APPROVED** [review]
Looks fantastic to me! I love the incorporation of thorough documentation and comments to help explain what's going on. I leave it to @mccauleyma to weigh in on the actual function of the code.

File: ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+unsigned char buf[1]; // memory for incoming message
+unsigned long canId; // CAN id of incoming message
+
+MCP_CAN CAN(SPI_CS_PIN); // setup can device
+
+/**
+ * Start CAN and intialize digital pin
+ */
+void setup() {
+  Serial.begin(115200); // baud rate for CAN
```
Comment (mccauleyma) [inline ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino:20]: Isn't this the baud rate for the serial output and nothing to do with CAN?

Comment (mccauleyma) [reply ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino:20]: Just wanna make sure the comment is right.

**Review by mccauleyma — CHANGES_REQUESTED** [review]
Based on overall skim, looks great to me. I'll leave the true functionality test to when we actually test it on the test rig
Just check that one comment plz.

File: ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino (hunk 1 above)
Comment (rydzynskip) [reply ArduinoCode/Driver-IO/BasicRelay/BasicRelay.ino:20]: Yes good catch. Baud rate is CAN.begin(CAN_500KBPS, ....

**Review by jamescd18 — APPROVED** [review]
Looking good!

**Review by mccauleyma — APPROVED** [review]
Looks good. You should be able to test this with what you have right?

**Comment by rydzynskip** [comment]
Yes just need two CAN modules and arduinos

**Pull request merged**